crates/ruff_python_ast/src/node.rs

```diff
@@ -2602,48 +2595,6 @@ impl AstNode for ast::ExprCall {
         visitor.visit_arguments(arguments);
     }
 }
-impl AstNode for ast::ExprFormattedValue {
```

Comment:
We'll still need the implementations for `FormattedValue` and `StringTodo`

Reply:
ok. what's the rule for when the "helpers" in ast::nodes should be considered nodes?

related: which of these should have methods in the visitor traits?

Reply:
bump @MichaReiser ?

Reply:
As a rule of thumb. They are nodes if
* they contain other nodes (is the case for `FormattedValue`)
* they are in a union with other nodes. The way I think about this is that there are nodes and the rest is mainly tokens/literals. We don't want unions where one variant is a terminal literal and another variant is anode. We either want unions over literals, or nodes. That's why `StringTodo` should be a node 
* If they have a range (`Identifier` is the exception to this)

We may not need dedicated visitor methods for `StringTodo` and `FormattedValue`, instead, we can create a single `visit_fstring_part` method.